packages/authhero/CHANGELOG.md

```
@@ -4,12 +4,19 @@
 ### Minor Changes
 
+<<<<<<< HEAD
+- 6858af2: Add client credentials scopes and permissions
+=======
 - 149ab91: Drop the old application table
+>>>>>>> main
 - b0e9595: Add client grants
 
 ### Patch Changes
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)